### PR TITLE
Adds uptime bot

### PR DESCRIPTION
**Pull request by bg451**
Uptime bot hits a bunch of pingdom checks, calculates the uptime for
each service, then posts it to slack.

### CODE REVIEW

**Comment by treeder** [comment]
@bg451 looks good, but need exact step by step in README for how to use it since I'm assuming it's not following the same directions as the rest: https://github.com/treeder/slackbots/blob/master/README.md

**Comment by treeder** [comment]
This ready to be merged?

**Comment by bg451** [comment]
Yeah its good to go

File: uptime/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  ]
+}
+```
+#### Run `go build` to build the binary.
+`./uptime` to run the program.
+
+## Uploading to iron worker
+#### Build the binary for amd64
+```
+GOOS=linux GOARCH=amd64 go build
```
Comment (treeder) [inline uptime/README.md]: Can you make this the dockerworker workflow?  ie: build it inside docker: https://github.com/iron-io/dockerworker/tree/master/go

File: uptime/pingdom.go [hunk 2]
```diff
@@ -0,0 +1,274 @@
+package main
```
Comment (treeder) [inline uptime/pingdom.go:1]: Is this basically an entire pingdom client?

Comment (bg451) [inline uptime/pingdom.go:1]: Just the stuff for the endpoints that actually show uptime numbers. The existing go pingdom client is missing half the api.

**Comment by bg451** [comment]
@treeder

**Pull request merged**